molgenis-metadata-manager/src/main/frontend/test/unit/specs/actions.spec.js

```diff
@@ -13,15 +13,11 @@ import {
   SET_SELECTED_ENTITY_TYPE_ID,
   UPDATE_EDITOR_ENTITY_TYPE
 } from 'store/mutations'
+
 import actions, { toAttribute, toEntityType } from 'store/actions'
 
 describe('actions', () => {
```

Comment:
mocha warns against use of arrow function style, https://mochajs.org/#arrow-functions